### ISSUE TITLE
Bug: setting strings converted to lowercase

### ISSUE THREAD

**Issue by hlugt** [qdm12/gluetun#1629]
### Is this urgent?

None

### Host OS

docker

### CPU arch

x86_64

### VPN service provider

IPVanish

### What are you using to run the container

docker-compose

### What is the version of Gluetun

3.34.2

### What's the problem 🤔

The auth.conf password is made lower case. Environment variable openvpn_password contains correct cased password. 
Fails to connect to vpn provider with authentication error. When manually correcting password in auth.conf authentication succeeds again, but ofcourse will get overwritten on new container.

Edit1: Think this changed recently: now notice that SERVER_COUNTRIES setting (on another instance) also fails due to being changed to lowercase version and thus not being recognized anymore...

Edit2: v3.34.1 works correctly.

### Share your logs

```plain text
ERROR [openvpn] AUTH: Received control message: AUTH_FAILED
Your credentials might be wrong 🤨
2023-05-30T17:54:15+02:00 INFO [openvpn] SIGUSR1[soft,auth-failure] received, process restarting
```

**Comment by qdm12** [comment]
That bug is a bug on the latest image, not v3.34.2, please double check the version you are using.

And the latest image got also fixed in 2c30984a10bd0667a715fbd5f77b268a58a0af42 you can just pull `qmcgaw/gluetun` if you want 😉

**Comment by hlugt** [comment]
Ok, you are correct, it is the latest image build that (still) fails: "Running version latest built on 2023-05-25T12:12:44.932Z (commit a43973c)"
Checked image tag v3.34.2 and that one is also working correctly.
(So fyi: still something wrong with "latest" pipeline)
